commit 45c2935e88d1eaf34c0769f9b514c0dcb0e43c1d
Author: William Cohen <wcohen@redhat.com>
Date:   Wed Jun 22 14:08:30 2016 -0400

    Correct IBM Power7 and Power8 computation of PAPI_L1_DCA
    
    When reviewing the test results for IBM Power7 and Power8 Michael
    Petlan found that the PAPI_L1_DCA preset was incorrectly computed.
    The L1 cache misses need to be subtracted rather than added to the
    result.
    
    Signed-off-by: William Cohen <wcohen@redhat.com>

diff --git a/src/papi_events.csv b/src/papi_events.csv
index 5c04442..2d2eca0 100644
--- a/src/papi_events.csv
+++ b/src/papi_events.csv
@@ -1349,7 +1349,7 @@ PRESET,PAPI_L1_LDM,NOT_DERIVED,PM_LD_MISS_L1
 PRESET,PAPI_L1_STM,NOT_DERIVED,PM_ST_MISS_L1
 PRESET,PAPI_L1_DCW,DERIVED_SUB,PM_ST_FIN,PM_ST_MISS_L1
 PRESET,PAPI_L1_DCR,DERIVED_SUB,PM_LD_REF_L1,PM_LD_MISS_L1
-PRESET,PAPI_L1_DCA,DERIVED_POSTFIX,N0|N1|+|N2|+|N3|+,PM_ST_FIN,PM_ST_MISS_L1,PM_LD_REF_L1,PM_LD_MISS_L1
+PRESET,PAPI_L1_DCA,DERIVED_POSTFIX,N0|N1|-|N2|+|N3|-,PM_ST_FIN,PM_ST_MISS_L1,PM_LD_REF_L1,PM_LD_MISS_L1
 PRESET,PAPI_L2_DCM,NOT_DERIVED,PM_DATA_FROM_L2MISS
 PRESET,PAPI_L2_LDM,NOT_DERIVED,PM_L2_LD_MISS
 PRESET,PAPI_L2_STM,NOT_DERIVED,PM_L2_ST_MISS
@@ -1398,7 +1398,7 @@ PRESET,PAPI_L1_LDM,NOT_DERIVED,PM_LD_MISS_L1
 PRESET,PAPI_L1_STM,NOT_DERIVED,PM_ST_MISS_L1
 PRESET,PAPI_L1_DCW,DERIVED_SUB,PM_ST_FIN,PM_ST_MISS_L1
 PRESET,PAPI_L1_DCR,DERIVED_SUB,PM_LD_REF_L1,PM_LD_MISS_L1
-PRESET,PAPI_L1_DCA,DERIVED_POSTFIX,N0|N1|+|N2|+|N3|+,PM_ST_FIN,PM_ST_MISS_L1,PM_LD_REF_L1,PM_LD_MISS_L1
+PRESET,PAPI_L1_DCA,DERIVED_POSTFIX,N0|N1|-|N2|+|N3|-,PM_ST_FIN,PM_ST_MISS_L1,PM_LD_REF_L1,PM_LD_MISS_L1
 PRESET,PAPI_L2_DCM,NOT_DERIVED,PM_DATA_FROM_L2MISS
 #n/aPRESET,PAPI_L2_LDM,NOT_DERIVED,PM_L2_LD_MISS
 #n/aPRESET,PAPI_L2_STM,NOT_DERIVED,PM_L2_ST_MISS
